aea/crypto/base.py

```diff
@@ -128,14 +137,60 @@ def sign_transaction(self, transaction: JSONLike) -> JSONLike:
         :return: signed transaction
         """
 
-    @abstractmethod
-    def dump(self, fp: BinaryIO) -> None:
+    @classmethod
+    def load(cls, private_key_file: str, password: Optional[str] = None) -> str:
```

Comment:
For symmetry introduced this here, also means framework is responsible for IO and not plugin.